resources/profiles/Geeetech/filament/Generic ASA @Geeetech.json

```diff
@@ -0,0 +1,17 @@
+{
+    "type": "filament",
+    "filament_id": "GFB98",
+    "setting_id": "GFSA04",
```

Comment:
Bambu Studio has strict setting_id rules. The format of setting id is to insert an S in the third position of filamnet id + GEEETECH + idx. For example, the filamnet id of ABS is GFB00, and the setting id should be GFB00_GEEETECH_00. The next ABS profile should be GFB00_GEEETECH_01. Setting ids cannot be repeated.

Reply:
Revised filament_id as requested